### ISSUE TITLE
[Regional Board of the Plast/Крайовий Провід Пласту] [Документообіг] Download buttons aren't named

### ISSUE THREAD

**Issue by NataliyaDominyuk** [ita-social-projects/EPlast#2388]
**Environment:** Mac OS Monterey 12.3.1, Google Chrome version 100.0.4896.127 (arm64)
**Reproducible:** always
**Build found:** 4ad6f9adcfc63c77d6ec449ec972f4889b535c3f
**Priority:** low
**Severity:** minor

**Preconditions**
Go to https://eplast.westeurope.cloudapp.azure.com/signin
Log in as a Plast in any role, except registered used

**Steps to reproduce**
1. Go to menu 'Regional Board of the Plast/Крайовий Провід Пласту'
2. Click on 'Документообіг'
3. Click on 'More/Більше' button 
4. Check the download button

**Actual result**
Download buttons aren't named (have no tags) which may cause accessibility issues

**Expected result**
Download buttons should have names (tags) for people with accessibility issues (for screen readers and other assistive technologies to announce it as button)
![Button name for better accessibility](https://user-images.githubusercontent.com/104838173/167108432-04d86f57-a27b-4423-b9a9-239cbaa2f8c4.jpg)
